data/parcel locker operators.tsv

```diff
@@ -3,4 +3,4 @@ Itella	Itella	Smartpost
 Venipak	Venipak
 DPD	DPD
 Unisend	Unisend	uDrop
-Latvijas Pasts	Latvijas Pasts
\ No newline at end of file
+Latvijas Pasts	Latvijas Pasts  Pasta Stacija   PastaStacija
```

Comment:
I changed the separators between entries to tabs, otherwise it didn't get parsed.

Reply:
Hmmm... I've put tabs there.
Probably VS code allowed to much authority to itself.

Thanks for catching